packages/010editor.vm/tools/chocolateyinstall.ps1

```diff
@@ -3,7 +3,7 @@ Import-Module vm.common -Force -DisableNameChecking
 
 try {
   $toolName = '010Editor'
-  $category = 'Hex Editors'
+  $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
```

Comment:
Documenting discussion: It is a good idea to edit two packages (common and an extra one) in this PR to test `VM-Get-Category`. But it is a good idea to split the update of other packages in separate PR (for example 30 packages per PR) so that the CI takes less to run and we do not encounter any issues when pushing the packages to MyGet.

Reply:
Note the issue with 010Editor (just in case you notice slow local testing or CI): https://github.com/mandiant/VM-Packages/issues/1275

Reply:
@Ana06 2 test packages have been removed from current PR